src/Plutarch/Extra/Maybe.hs

```diff
@@ -151,3 +156,23 @@ pexpectJustC escape ma = tcont $ \f ->
     pmatch ma $ \case
         PJust v -> f v
         PNothing -> escape
+
+{- | Extract the value stored in a PMaybe container.
+     If there's no value, throw an error with the given message.
+     @since 1.3.0
+-}
+mustBePJust :: forall a s. Term s (PString :--> PMaybe a :--> a)
```

Comment:
I think we could use better naming conventions with the other functions in this module (like `pfromDJust`.)

Reply:
And likewise similar interfaces (passing in the error string vs not)

Reply:
The intention here is to show a error message. I am not sure if this should be upstreamed to be honest.

Reply:
I renamed it to `passertPJust` and `passertPDJust`